ansible/roles/debops.docker/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: Get upstream APT GPG key
+  apt_key:
+    id: '{{ docker__upstream_key }}'
+    keyserver: '{{ ansible_local.core.keyserver
+                   if (ansible_local|d() and ansible_local.core|d() and
+                       ansible_local.core.keyserver)
+                   else "hkp://pool.sks-keyservers.net" }}'
+    state: 'present'
+  register: docker__register_apt_key
+  until: docker__register_apt_key is succeeded
+
+- name: Configure upstream APT repository
+  apt_repository:
+    repo: '{{ docker__upstream_repository }}'
+    mode: '0644'
+    state: 'present'
+    update_cache: True
```

Comment:
Can you drop the abolve two tasks and switch to using the `debops.keyring` role? The `debops.docker_server` has a good example on how to do this, created via https://github.com/debops/debops/commit/85c1ddf7270f50e860a2049431a8040547f72e04 commit.